main/libaio/APKBUILD

```diff
@@ -1,12 +1,12 @@
 # Contributor: Leonardo Arena <rnalrd@alpinelinux.org>
 # Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
 pkgname="libaio"
-pkgver=0.3.110
-pkgrel=1
+pkgver=0.3.111
+pkgrel=0
 pkgdesc="Asynchronous input/output library"
 url="http://lse.sourceforge.net/io/aio.html"
 arch="all"
-license="LGPL-2.0"
+license="LGPL-2.1-or-later"
```

Comment:
On a quick first look i didn't find an 'or-later' clause in the source code. It's definitely licensed under LGPL-2.1 though.

Reply:
The included license file indicates the `*-or-later` variant :

    This library is free software; you can redistribute it and/or
    modify it under the terms of the GNU Lesser General Public
    License as published by the Free Software Foundation; either
    version 2 of the License, or (at your option) any later version.

Reply:
Ah! I only looked at the C source files.